hooks/post_gen_project.py

```diff
@@ -55,7 +55,7 @@ def remove_filepath(filepath):
 
 
 cicd_specific_paths = {
-    "gitHub": [
+    "git": [
```

Comment:
Could you explain purpose of the changes in this file?

Reply:
`gitHub` and `gitHubEnterprise` both uses `.github` directory, that python cannot hash lists or sets as keys so I make a common key for both `gitHub` and `gitHubEnterprise`, the reason not to hash both `gitHub` and `gitHubEnterprise` is because of the below code blocks where we remove file paths of unwanted directories.

Reply:
Got it. It's the unrelated cleanup logic after the project generation. You are matching the prefix to map `gitHub` and `gitHubEnterprise` to the same entity in the map.

Since there are only 2 options, how about Removing the map and explicitly checking the condition
`cicd_platform == "gitHub"/"gitHubEnterprise"/"azureDevOpsServices"` and remove related files?

In this way there's no implicit cicd_platform name assumption.

Reply:
Agree, the remove_map logic is kinda weird, I will change to explicitly check condition.